LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 sulaimangari
```

Comment:
Its not really appropriate to assign your own copyright to someone else's project.

Reply:
my bad, I forgot to change it so the license refer to you